Update README (#33)
blockchaintp/daml-on-sawtooth: README.md

NOTE: By default, it will start `daml-on-sawtooth` with `AuthService` switched off.

> ### 4. Interacting with daml-on-sawtooth using daml assists

Reviewer: ```suggestion
### 4. Interacting with daml-on-sawtooth using daml assistant
```

* Open a terminal (one where you are not hosting a running daml-on-sawtooth see STEP 3).

> * Create a [sample daml project](https://docs.daml.com/getting-started/quickstart.html), if you do not already have one.

Reviewer: ```suggestion
* If you haven't already done so, create a [sample daml project](https://docs.daml.com/getting-started/quickstart.html).
```
